mlrun/api/utils/clients/log_collector.py

```diff
@@ -191,3 +195,39 @@ async def has_logs(
                     f"{msg},error= {response.errorMessage}"
                 )
         return response.hasLogs
+
+    async def stop_logs(
+        self,
+        project_to_run_uids_dict: dict,
+        verbose: bool = False,
+    ) -> None:
+        """
+        Stop logs streaming from the log collector service
+        :param project_to_run_uids_dict: a dict of run uids per project to stop logs for
+        :param verbose: Whether to log errors
+        :param raise_on_error: Whether to raise an exception on error
+        :return: None
+        """
+        # convert the dict to a map with protobuf StringArray
+        request_dict = {}
+        for project, runs in project_to_run_uids_dict.items():
+            request_dict[project] = self._log_collector_pb2.StringArray(
+                values=project_to_run_uids_dict[project]
+            )
+
+        request = self._log_collector_pb2.StopLogRequest(
+            projectToRunUIDs=request_dict,
+        )
+
+        try:
+            # StopLogs has no return value, so we don't need to check the response
+            await self._call("StopLog", request)
```

Comment:
what about a response status? such as we have in http (OK, Accepted etc...) we would like to check that the log collector received it